lib/notifications/export_templates.rb

```diff
@@ -0,0 +1,36 @@
+require "notifications/client"
+require "./lib/common/gateway/s3_object_uploader"
+require "./lib/services"
+
+module Notifications
+  class ExportTemplates
+    TEMPLATE_ATTRIBUTES = %w[
+      id
+      type
+      name
+      subject
+      body
+      letter_contact_block
+      version
+      created_at
+      updated_at
+      created_by
+    ].freeze
+
+    def self.execute
+      body = StringIO.new(templates.to_json)
```

Comment:
I don't think you need StringIO, you can just use a string in the upload method. The templates are already in memory and don't need to be streamed.